geoindicators/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/geoindicators/SpatialUnits.groovy

```diff
@@ -432,4 +434,40 @@ IProcess spatialJoin() {
             [outputTableName: outputTableName, idColumnTarget: idColumnTarget]
         }
     }
+}
+
+/**
+ * This process is used to generate a regular grid in meters.
+ *
+ * @param geometry A geometry defined as Point, Line or Polygon
+ * @param deltaX A integer that represents the horizontal step of a ceil in the mesh
+ * @param deltaY A integer that represents the vertical step of a ceil in the mesh
+ * @param outputTable A Table that contains the geometry defining the regular grid
+ * @param datasource A connexion to a database (H2GIS, PostGIS, ...) where are stored the input Table and in which
+ *        the resulting database will be stored
+ * @param outputTableName The name of the created table
+ * */
+IProcess regularGrid() {
+    return create {
+        title "Creating a regular grid in meters"
+        id "regularGrid"
+        inputs geometry: Geometry, deltaX: int, deltaY: int, outputTable: String, datasource: JdbcDataSource
+        outputs outputTableName: String
+
+        run { geometry, deltaX, deltaY, outputTable, datasource ->
+            if (datasource.hasTable("$outputTable")) {
```

Comment:
Instead of `datasource.hasTable("$outputTable")` you can write  `datasource.hasTable(outputTable)`